assets/scss/_footer.scss

```diff
@@ -23,26 +18,32 @@
     @include unstyled-list;
   }
 
+  h2,
   a {
+    font-size: 13px;
```

Comment:
I missed this, I think our font-sizes are supposed to be in rem, so they resize with user default changes.

Reply:
Thanks for catching this. I must have copied this from the browser inspector's calculated value instead of the supplied value in https://github.com/GSA-TTS/identity-site/

Reply:
Here's the source: 

https://github.com/GSA-TTS/identity-site/blob/a2a42a1d00d1d58dc5c45c5efaa4cbf7aba00898/_sass/_variables.scss#L4-L5

Reply:
I was under the impression that there's a potential problem with rem and font sizes where, since rem are relative, certain contexts can cause the same rem font sizes to be different actual sizes of font within the same page. That said

* we clearly use it regularly in other code bases
* I like consistency
* I would not be surprised if our CSS standard practices are such that the potential problem is guaranteed never to arise

I'll change this to 0.85rem, since that seems to be the closest multiple of 0.05rem

Reply:
if you use `em`, you risk having 1em being different px sizes on the same page, because that is relative to the "m" character of the font you're currently using at the current font size. So, if I have a `div` set to 16px, then a `p` set to 1.5em, that's 24px; if that `p` has a `span` that's 1.5em, the size of an "m" on the parent is 24px, so the span text is 36px.

`rem` is "root em", which is relative to the font-size of the root, in the case of an HTML document, the `html` element. Because that is static throughout the document, 1.5em is always the same px. The trick here is that users can adjust the root size in their browser settings (or with extensions), which then causes every element's font-size to scale the same amount, which makes it very useful for allowing users to "zoom in" the text without having to scroll sideways as they would if they zoomed the whole page.

Reply:
the `ex` unit is relative to the height of the "x" character, which is not often used in Web development but when you need it, is a fantastic trick.

Reply:
Ah! So in concern about this potential problem, I was getting `em` and `rem` switched. Got it. Unsure where I got the idea that it was "relative em" instead of "root em"